openeo/service.py

```diff
@@ -1,24 +1,44 @@
 from abc import ABC
-
+from typing import List
```

Comment:
re: https://github.com/Open-EO/openeo-python-client/issues/115#issuecomment-832515073

I think it's better to remove openeo/service.py
and implement this new stuff under openeo/rest/service.py, because it depends on a working rest Connection

Reply:
I just removed openeo/service.py in 6b9318d87c0

Reply:
I've moved it to rest, but should it also get the "REST" prefix in filename and class name?

Reply:
I wouldn't add that, I'm not a fan of that REST prefix